google-cloud-firestore/src/main/java/com/google/cloud/firestore/DocumentReference.java

```diff
@@ -354,6 +354,19 @@ public T apply(List<T> results) {
     return extractFirst(firestore.getAll(this));
   }
 
+  /**
+   * Reads the document referenced by this DocumentReference. If the document doesn't exist, the
+   * get(FieldMask fieldMask) will return an an empty DocumentSnapshot.
+   *
+   * @param fieldMask A FieldMask Object for retrieve field value.
+   * @return An ApiFuture that will be resolved with the contents of the Document at this
+   *     DocumentReference, or a failure if the document does not exist.
+   */
```

Comment:
the contents -> the content
**@param** <txt>. -> @param <txt>     (no . is required)
same for **@return**

Reply:
Done